python/lib/scheduler/dmod/scheduler/job/job_util.py

```diff
@@ -40,17 +40,9 @@ def factory_create(cls, **kwargs):
         JobManager
             A newly created ::class:`JobUtil` object.
         """
-        host = None
-        port = None
-        pword = None
-        for key, value in kwargs.items():
-            if key == 'redis_host':
-                host = value
-            elif key == 'redis_port':
-                port = int(value)
-            elif key == 'redis_pass':
-                pword = value
-        return RedisBackedJobUtil(redis_host=host, redis_port=port, redis_pass=pword)
+        return RedisBackedJobUtil(redis_host=kwargs.get('redis_host'), 
```

Comment:
Do these need default `None` or is it expected to raise a `keyerror` if these key's are not in `kwargs`